src/Metric.php

```diff
@@ -158,16 +164,12 @@ public function getHelpText($machine_name)
      */
     public function getResults($uri)
     {
-        $validator = new \Services_W3C_HTMLValidator(array(
-            'validator_uri' => $this->options['service_url']
-        ));
-
-        $request = new \HTTP_Request2();
-        $request->setConfig('adapter', 'HTTP_Request2_Adapter_Curl');
-        $validator->setRequest($request);
-        
-        $result = $validator->validate($uri);
+        $validator = new Validator($this->options['service_url']);
```

Comment:
Good question.  The library uses Guzzle, which is pretty magical in which handler it decides to use, but it has been using CURL for these requests, which will work.